index.html

```diff
@@ -254,12 +254,22 @@
   <p>The application can collect, process, and act on the provided metrics via the provided JavaScript interface:</p>
 
   <pre class="example js">
-  window.performance
-    .getEntriesByName('https://example.com/resource.jpg', 'resource')
-    .serverTiming
```

Comment:
The original example was wrong here.  `getEntriesByName()` returns a sequence, not an object.